php/aboutMeFunctions.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+* This Function Fetches Data from the database and returns the query as an associative array.
+*
+* @param the $db is to input the new instance of PDO database.
+*
+*
+* @return Assoc array which contains results.
```

Comment:
@return missing variable after datatype and does not explain what variable represents

Reply:
done